python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -435,6 +435,101 @@ def convert_pad(node, **kwargs):
     return [node]
 
 
+@mx_op.register("dot")
+def convert_dot(node, **kwargs):
+    """Map MXNet's dot operator attributes to onnx's
+    MatMul and Transpose operators based on the values set for
+    transpose_a, transpose_b attributes."""
+    helper, _, _ = import_onnx_modules()
+    proc_nodes = kwargs["proc_nodes"]
+    node_inputs = node["inputs"]
+    name = node["name"]
+
+    input_a_idx = kwargs["index_lookup"][node_inputs[0][0]]
+    input_node_a = proc_nodes[input_a_idx].name
+    input_b_idx = kwargs["index_lookup"][node_inputs[1][0]]
+    input_node_b = proc_nodes[input_b_idx].name
+
+    # Getting the attributes and assigning default values.
+    if "attrs" in node:
```

Comment:
Can use two lines for these block:
trans_a = int(node.get('attrs', {}).get('transpose_a', 0))
trans_b = int(node.get('attrs', {}).get('transpose_b', 0))

I am not sure if the transpose_a is encoded as string '1' or 'True', if the latter, then it's not valid

Reply:
Thanks! Good catch. Seems like mxnet allows both "1" and "True", made changes accordingly.